### DISCUSSION TITLE
Dating Gregorian vs. Julian Calendar

### DISCUSSION THREAD

**Discussion by freedmandil** [Stellarium/stellarium#3911] [Ideas]
### Using Different Calendar Systems in Different Periods/Epochs

First of all, I'm not talking about the JD date that astronomers use for dating,
I'm talking about the Julian Calendar vs. the Gregorian Calendar
In terms of historical dating, and measuring periods of time, is there a way of ensuring consistency in the Extrapolated Gregorian Calendar - meaning March 21 remains the vernal equinox in the distant past vs today rather than switching to the Julian calendar before 1582 for Catholic Countries, and 1752 for Commonwealth countries and other dates for the Russians and French Calendars.  Being able to decide which calendar to use, during which epoch would be tremendously useful rather than having to rely on the JD number.

**Comment by alex-w** [comment]
Please use Calendars plugin

**Comment by gzotti** [comment]
And make sure to have read Appendix F of the User Guide before using the program for historical applications.
